includes/class-freemius.php

```diff
@@ -509,23 +547,69 @@ private function _register_account_hooks() {
 		 * @uses   fs_find_caller_plugin_file
 		 */
 		private function _find_caller_plugin_file() {
-			// Try to load the cached value of the file path.
+			return $this->_storage->plugin_main_file->path;
+		}
+
+
+		/**
+		 * Identifies the caller type: plugin or theme.
+		 *
+		 * @author Leo Fajardo (@leorw)
+		 * @since  1.2.0
+		 *
+		 * @return string
```

Comment:
@vovafeldman Sorry, could you please clarify? The method returns the caller module's path and this line in the constructor retrieves and saves the module's path `$this->identify_caller_main_file_and_type();`.